test/acceptances/http_test.exs

```diff
@@ -0,0 +1,44 @@
+defmodule Acceptances.HTTPTest do
+  use ExUnit.Case
+
+  import Tirexs.HTTP
+
+  @uri_environment Application.get_env(:tirexs, :uri)
+
+
+  setup do
+    delete("bear_test") && :ok
```

Comment:
have you ever used `on_exit` teardown in `ExTest`?
I am asking because I was unable to make it work properly.
Here we are implicitly  relying on the fact that `put` will create the elasticsearch index I think
In some setups index creation with `put` is disabled AFAIK
check here for doc https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html#index-creation
Use case for a user would be to have an error, because the user *wants* to put mappings and having indices explicitly created.

However let's keep in mind this, I think it's a general thing to address separately